src/components/ButtonPanel.js

```diff
@@ -2,37 +2,39 @@ import React from 'react';
 import Button from './Button';
 import '../styles/ButtonPanel.scss';
 
+const lightorange = '#ffa64d';
+
 export default function ButtonPanel() {
   return (
     <div className="buttonPanel">
       <div className="groupOne">
-        <Button name="AC" />
-        <Button name="+/-" />
-        <Button name="%" />
-        <Button name="÷" />
+        <Button wide={false} name="AC" />
+        <Button wide={false} name="+/-" />
+        <Button wide={false} name="%" />
+        <Button wide={false} color={lightorange} name="÷" />
       </div>
       <div className="groupTwo">
-        <Button name="7" />
-        <Button name="8" />
-        <Button name="9" />
-        <Button name="X" />
+        <Button wide={false} name="7" />
+        <Button wide={false} name="8" />
+        <Button wide={false} name="9" />
+        <Button wide={false} color={lightorange} name="X" />
       </div>
       <div className="groupThree">
-        <Button name="4" />
-        <Button name="5" />
-        <Button name="6" />
-        <Button name="-" />
+        <Button wide={false} name="4" />
+        <Button wide={false} name="5" />
+        <Button wide={false} name="6" />
+        <Button wide={false} color={lightorange} name="-" />
       </div>
       <div className="groupFour">
-        <Button name="1" />
-        <Button name="2" />
-        <Button name="3" />
-        <Button name="+" />
+        <Button wide={false} name="1" />
+        <Button wide={false} name="2" />
+        <Button wide={false} name="3" />
+        <Button wide={false} color={lightorange} name="+" />
       </div>
       <div className="groupFive">
-        <Button name="0" />
-        <Button name="." />
-        <Button name="=" />
+        <Button wide name="0" />
+        <Button wide={false} name="." />
+        <Button wide={false} color={lightorange} name="=" />
       </div>
     </div>
   );
```

Comment:
[Optional]
You can refactor the code. Think about arrays containing your button symbols.